src/scripts/main.js

```diff
@@ -1 +1,104 @@
 'use strict';
+
+function promise1() {
+  const resolver = (confirm1, reject1) => {
```

Comment:
name these variables properly everywhere, without numbers

```suggestion
  const resolver = (resolve, reject) => {
```